apps/src/code-studio/components/ShareAllowedDialog.jsx

```diff
@@ -372,14 +372,10 @@ class ShareAllowedDialog extends React.Component {
                   </div>
                   <div>
                     <Button
-                      color={Button.ButtonColor.brandSecondaryDefault}
+                      color={buttonColors.purple}
+                      type="primary"
                       id="sharing-dialog-copy-button"
-                      icon="clipboard"
-                      style={{
-                        ...styles.button,
-                        ...styles.copyButton,
```

Comment:
Can we remove this styling now?

Reply:
Thanks for the catch! We still do need `styles.button`, but `.copyButton` indeed can be removed. (making this update now)